sonic_data_client/mixed_db_client.go

```diff
@@ -75,6 +75,87 @@ type MixedDbClient struct {
 
 var mixedDbClientMap = map[string]MixedDbClient{}
 
+func hget(table swsscommon.Table, key string, field string) (string, error) {
+	var fieldValuePairs = swsscommon.NewFieldValuePairs()
+	var result = table.Get(key, fieldValuePairs)
+	if result {
+		var fieldCount = int(fieldValuePairs.Size())
+		for idx := 0; idx < fieldCount; idx++ {
+			var pair = fieldValuePairs.Get(idx)
+			if pair.GetFirst() == field {
+				return pair.GetSecond(), nil
+			}
+		}
+	}
+
+	return "", fmt.Errorf("Can't read %s:%s from %s table", key, field, table.GetTableName())
+}
+
+func getDpuAddress(dpuId string) (string, error) {
+	var configDb = swsscommon.NewDBConnector("CONFIG_DB", SWSS_TIMEOUT, false)
+
+	// Find DPU address by DPU ID from CONFIG_DB
+	// Design doc: https://github.com/sonic-net/SONiC/blob/master/doc/smart-switch/ip-address-assigment/smart-switch-ip-address-assignment.md?plain=1
+
+	// get bridge plane
+	var bridgeTable = swsscommon.NewTable(configDb, "MID_PLANE_BRIDGE")
```

Comment:
>NewTable [](http://example.com/codeflow?start=30&length=8)

Is it possible to use ConfigDBConnector? I understand we may need a new code into swss-common to support using it in golang.

Reply:
Will check if can use get_entry() method in ConfigDBConnector.